doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -39,7 +39,7 @@ The nRF54H20 DK uses multiple board files for its configuration:
 * `Risc-V cores configuration files`_
 
 You can use these files as a starting point for configuring your own custom board.
-The easiest way to do so, when creating a :ref:`Zephyr repository application <zephyr:zephyr-repo-app>`, is to create a copy of these folders under :file:`sdk-nrf-next/boards/arm/your_custom_board_name` (for the ARM configuration files) and :file:`sdk-nrf-next/boards/riscv/your_custom_board_name` (for the Risc-V configuration files), respectively.
+The easiest way to do so, when creating a :ref:`Zephyr repository application <zephyr:zephyr-repo-app>`, is to create a copy of these folders under :file:`boards/arm/your_custom_board_name` (for the ARM configuration files) and :file:`boards/riscv/your_custom_board_name` (for the Risc-V configuration files), respectively.
```

Comment:
`boards/arch` no longer exist, it's now `boards/vendor`.

Reply:
@gmarull updated

Reply:
@gmarull updated